styles.css

```diff
@@ -0,0 +1,38 @@
+body {
+  background: navy;
+  color: #fff;
+}
+
+#survey-form {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin: 2rem auto;
+}
+
+
+.group-dropdown {
+  width: 100%;
+}
+
+label,
+input,
+textarea {
+  padding: 1rem 0;
+}
+
+input,
+textarea {
+  font-size: 1.25rem;
+  padding-left: 1rem;
+}
+
+.center {
+  width: 80%;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.title {
+  color: aliceblue;
+}
```

Comment:
wow, you have really done well but it just remains a little to make your code professional and dry by implementing those things I appointed in my review.

And also endeavor to add **linters** to make your code more dry and professional, it helps fetch out errors in your code and make it more standard. :)

Cheers :) Happy Coding.